examples/dreambooth/requirements.txt

```diff
@@ -4,3 +4,4 @@ transformers>=4.25.1
 ftfy
 tensorboard
 Jinja2
+peft>=0.6.0
```

Comment:
Let's also add to https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/requirements_sdxl.txt. Applies for text-to-image, too.